src/main/kotlin/com/team4099/robot2023/subsystems/superstructure/Superstructure.kt

```diff
@@ -1167,6 +1168,24 @@ class Superstructure(
     return returnCommand
   }
 
+  fun testDriveVelocityCommand(): Command {
+    val returnCommand = runOnce {
+      currentRequest= Request.SuperstructureRequest.Tuning()
+      drivetrain.currentRequest =
```

Comment:
don't set requests of DT from superstructure. that's an unsafe operation

Reply:
this is because there are commands that are simultaneously setting requests as this command which are running in parallel. so the subsystem will try to do both and then something goofy will happen.

Reply:
adding onto this, the addRequirements function i showed yall the other day does exactly this. since we don't specify dt as a "requirement" when running superstructure, it lets you make operations to dt from both dt commands which specify that they require the subsystem and from superstructure, which has nothing about requiring dt, causing stuff to go wrong.

Reply:
deleted bc unused 👍